src/tbp/monty/frameworks/config_utils/make_env_interface_configs.py

```diff
@@ -91,30 +91,24 @@ class SupervisedPretrainingExperimentArgs(ExperimentArgs):
 # Data-set containing RGBD images of real-world objects taken with a mobile device
 @dataclass
 class EnvInitArgsMontyWorldStandardScenes:
-    data_path: str = os.path.join(
-        os.environ["MONTY_DATA"], "worldimages/standard_scenes/"
-    )
+    data_path = Path(os.environ["MONTY_DATA"]) / "worldimages/standard_scenes/"
```

Comment:
Same here with using a `/` in the string instead of the operator. And below as well.